READMEs/c2d-flow.md

```diff
@@ -290,13 +319,24 @@ Here is a list of possible results: [Operator Service Status description](https:
 Once you get `{'ok': True, 'status': 70, 'statusText': 'Job finished'}`, Bob can check the result of the job.
 
 ```python
-result = ocean.compute.result(DATA_did, job_id, 0, bob_wallet)  # 0 index, means we retrieve the results from the first dataset index
+# Retrieve algorithm output and log files
+for i in range(len(status["results"])):
```

Comment:
I'd rather not add this here. We want to encourage users to explore the results and understand them. We should show them samples of the result, and have them maybe identify the index visually. At least that is my opinion, I think it would provide a richer experience and understanding, not just mechanical copy-pasting. It depends on what we want to achieve: do we want a flow where users get to know Ocean, or one where copy-pasting everything makes it "go"? @trentmc I need your opinion here as well.

Reply:
Summary: I recommend that we keep this part of the PR.

Details...

> do we want a flow where users get to know Ocean, or one where copy-pasting everything makes it "go"

Ideally we have something that gets both. 

The README now does both here. How:

- The code under "# Retrieve algorithm output and log files" shows how to make it go
- The code under "You can use the result however you like. For the purpose of this example, let's plot it" helps to promote understanding